website/components/navigation/Navigation.tsx

```diff
@@ -82,11 +82,13 @@ function Content({ isTouchDevice = false }: { isTouchDevice?: boolean }) {
     const isGuideline = type === GroupType.Guideline;
     const items = isGuideline ? coreGuidelines : components;
 
+    const collapsibleProp = isTouchDevice && { collapsible: true } as const
```

Comment:
spreading a boolean seems sketchy. why not `isTouchDevice ? { collapsible: true } : {};`

Reply:
👍